AirCasting/SessionViews/SessionCellView.swift

```diff
@@ -27,7 +28,10 @@ struct SessionCellView: View {
             thresholds: Array(thresholds))
             if !isCollapsed {
                 VStack(alignment: .trailing, spacing: 40) {
-                    pollutionChart
+                    #warning("The stream should be chosen based on users selection")
+                    if let streams = session.measurementStreams {
```

Comment:
Maybe we can have a helper method like in measurement:
```
 public var allMeasurements: [MeasurementEntity]? {
        measurements?.array as? [MeasurementEntity]
    }
```
to get the list of allMeasurementStreams where the `as!` would not be needed.